src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -58,7 +58,7 @@ public void parseIndex_validInput_success() throws Exception {
 
     @Test
     public void parseName_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> ParserUtil.parseName((String) null));
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseName(null));
```

Comment:
Just curious, why did you remove the `(String)` typecasting?

Reply:
Can you take a look at my new PR?

Reply:
If that PR is good then can merge it first, so that there won't be too many commits in a PR